src/commands/compile/compile.ts

```diff
@@ -1,5 +1,5 @@
 import path from 'path'
-import { getProgramFolders } from '../../utils/config'
+import { getProgramFolders, getMacroFolders } from '../../utils/config'
```

Comment:
Same function should also be used in `compileSingleFile.ts`.

Reply:
fixed